### DISCUSSION TITLE
buddhism/2024/buddha_eyes/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#125] [General]
# buddhism/2024/buddha_eyes/

http://shanenull.com/buddhism/2024/buddha_eyes/

**Comment by shane0** [comment]
# adamantine eyes

The eyes of Buddha, also known as Wisdom eyes, are a significant symbol in Buddhist art. They are typically depicted as two half-closed eyes, often referred to as the Adamantine View. Between and slightly above the eyes is a circle or spiral called the urna, and directly below the urna is a curly symbol stylized as १, which represents the number one in Devanagari numerals.

Here's a breakdown of the meanings of these symbols:

* **The two half-closed eyes:** These represent the Buddha's omniscience or all-seeing nature. They symbolize his ability to see all things in all directions and in all time.
* **The urna:** This circle or spiral is often depicted as a third eye between the Buddha's eyebrows. It represents the Buddha's spiritual wisdom and enlightenment.
* **The curly symbol:** This symbol, which looks like a question mark, represents the Nepali character for the number one. It symbolizes unity of all things and the one way to reach enlightenment through the Buddha's teachings.

These symbols are often found on stupas, Buddhist shrines, and statues, and they are believed to represent the Buddha's wisdom, enlightenment, and the path to spiritual liberation.

**10 Links for Citations on Buddha Eyes**

1. **The Metropolitan Museum of Art:** https://www.metmuseum.org/toah/hd/budd/hd_budd.htm
2. **The British Museum:** https://www.britishmuseum.org/collection/object/A_1899-0715-1
3. **The Smithsonian Institution:** https://www.si.edu/spotlight/encountering-the-buddha
4. **The Getty Museum:** https://www.metmuseum.org/toah/hd/budd/hd_budd.htm
5. **The Rubin Museum of Art:** https://rubinmuseum.org/
6. **The Encyclopedia of Buddhism:** https://www.britannica.com/topic/Buddhism
7. **The Buddhist Dictionary:** http://www.buddhism-dict.net/ddb/
8. **The Journal of Asian Studies:** https://www.jstor.org/journal/jasianstudies
9. **The Art Bulletin:** https://www.tandfonline.com/toc/rcab20/current
10. **The Journal of Buddhist Studies:** https://www.jbspress.com/
